### ISSUE TITLE
WiiU completeley freezes after a few seconds

### ISSUE THREAD

**Issue by Atramentius** [GaryOderNichts/moonlight-wiiu#28]
WiiU:
- Connected via WIFI
- Tiramisu and homebrew latest version
- Moonlight WiiU port Version 1.5
- stream quality 720p 30FPS
- bitrate 5000
- package size 1024

HostPC: 
- Connected via Ethernet
- Windows 11 Pro
- GFE Version 3.26.0.160
- Windows Firewall disabled for Shield Stream
- RTX2070S
- Driver 527.56
- AMD Ryzen 5 3600
- 16GB RAM

Upon starting the stream Steam starts in Big Picture mode. Navigating the Steam "Home" menu works just fine without any latency.
Opening the Steam Library works as well but a few seconds later on this screen, the Console Freezes with no way to exit to the WiiU Home Menu. I have to forcequit the entire console by pressing the power button for about 4sec.

Sometimes the stream ends abbruptly and it drops back to the moonlight menu with an error: stream terminated. error: -1
I can connect back to my PC after this but then after a few seconds The console completely freezes again

On the host PC it says: Steam can't be shut down unless all games have been quit; but there weren't any games running in the first place.

On a previous issue they said installing the moonlight version 1.2 worked just fine, but I probably have to downgrade Geforce experience for that as well. 

Any way to fix my issue on the 1.5 moonlight version?

**Comment by Atramentius** [comment]
Ok I downgraded to GFE 3.26.0.131 which allowed me to start up Dark Souls 3 and also play it without a lot of problems but when I loaded my Elden Ring Save the Console froze again.

**Comment by RealTadango** [comment]
I use Sunshine as host software and it freezes to after a short time playing. Ofter it disconnects first and when i reconnect the Wii U freezes completely

**Comment by Atramentius** [comment]
> I use Sunshine as host software and it freezes to after a short time playing. Ofter it disconnects first and when i reconnect the Wii U freezes completely

Yes exact same problem. haven't figured out yet how to fix this issue. Im just waiting for a reply from GaryOderNichts or an update for Moonlight.

**Comment by RealTadango** [comment]
I think slow wifi is an issue. A low bitrate works fine. I ordered an ethernet adapter to test.

**Comment by GaryOderNichts** [comment]
Sorry for the late response, I've been busy over the past few weeks.
There seems to be an issue with reconnecting after several disconnects, causing a freeze.
Make sure to lower your setting to ensure a stable connection.

**Comment by Atramentius** [comment]
Okay well connection shouldn't be an issue as I can stream my Games to moonlight on an Ipad via Wifi @1080p60 just fine without any hickups; about 40ms latency maximum. The WiiU is on the exact same Wifi Network. I think The WiiU also crashes even if I don't have to reconnect after a few minutes at best.

**Comment by RealTadango** [comment]
I tried an ethernet adapter, but the same issue. After a minute of 3 the screen freezes but audio and control still work. On a low resolution and bitrate it all works fine 2mbit works, 6mbit crashes after a few minutes.

**Comment by All-StarsPixel** [comment]
ive set my bitrate to the lowest it can be (0.5) and the stream crashes imediately, then when reconnecting it freezes instantly. Also, steam doesnt open. that might be the issue so I will test it later. I'm using it with a steam deck for better ds, 3ds and wii u emulation. I guess it could be useful for other things

**Comment by GaryOderNichts** [comment]
There was an issue where the console froze after reconnecting several times. This has been fixed in [version 1.6](https://github.com/GaryOderNichts/moonlight-wiiu/releases/tag/wiiu-v1.6).
